doc/plugins/proxy-rewrite-cn.md

```diff
@@ -30,6 +30,7 @@
 |host            |可选| 转发到上游的新 `host` 地址，例如：`iresty.com` |
 |enable_websocket|可选| 是否启用 `websocket`（布尔值），默认不启用|
 |headers         |可选| 转发到上游的新`headers`，可以设置多个。头信息如果存在将重写，不存在则添加。想要删除某个 header 的话，把对应的值设置为空字符串即可|
+|resp_headers    |可选| 指定要输出出给客户端的Header，可以设置多个。头信息如果存在将重写，不存在则添加。|
```

Comment:
why `proxy-rewrite` need to modify response headers? Is it should only care about upstream?

Reply:
in my opinion, the plugin `proxy-rewrite` here is offering rewriter function in the proxy server, both rewrite data to upstream and from upstream , it used to be modify header to upstream, now I add a rewirite header function to modify header to client, it still kind of `rewrite function`

Reply:
`rewrite` and `proxy_pass` are only for upstream in Nginx, not downstream.
So I think `response header rewrite` should not in this plugin.

Reply:
compare to a independent CORS plugin, here we can set any customized header and it's more flexible, like add a server API  version header. or cluster id header

Reply:
OK  , I think I will take your idea,add a new plugin is also a better idea